### DISCUSSION TITLE
What is the benefit of using this over just plain Semantic-UI?

### DISCUSSION THREAD

**Discussion by MattLawz** [fomantic/Fomantic-UI#2328] [General] [answered]
The title says it all, is there a reason this was created or a reason people should use this over Semantic? Thanks in advance.

**Comment by ivangomes** [comment, marked as answer]
Semantic UI: Last Version 2.4.1 - Oct 13, 2018
Fomantic UI: Last Version 2.8.8 - Jun 24, 2021

the benefit of collecting the fruits of the hard and good work the Fomantic UI community has been doing to keep it in a good shape and fix the bugs and add more updates.

**Comment by lubber-de** [comment]
We also provide a nightly build via [NPM](https://www.npmjs.com/package/fomantic-ui/v/nightly) or [CDN](https://www.jsdelivr.com/package/npm/fomantic-ui?version=nightly) out of the current develop branch, which contains all the latest merged features and bugfixes meant as a beta build for the upcoming 2.9.0

So do not only rely on the latest official build timestamp to rate activity 😉 
We are already ~700 bugfixes and new features ahead of SUI

**Comment by MattLawz** [comment]
Thank you guys for the speedy answers, ill be moving to Fomantic-UI soon. 😀
